modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -355,6 +362,21 @@ public void setUp() throws Exception {
 					"localizedDateObjectFieldName"
 				).build(),
 				new DateTimeObjectFieldBuilder(
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).localized(
+					true
+				).name(
+					"localizedDateTimeObjectFieldName"
+				).objectFieldSettings(
+					Collections.singletonList(
+						_createObjectFieldSetting(
+							ObjectFieldSettingConstants.NAME_TIME_STORAGE,
+							ObjectFieldSettingConstants.
+								VALUE_USE_INPUT_AS_ENTERED))
+				).build(),
+				new DateTimeObjectFieldBuilder(
```

Comment:
You added the `localizedDateTimeObjectFieldName` before `dateTimeObjectFieldName`. I think you should sort.

Reply:
You're right; I fixed it. I believe the correct order is to have the name of the builder class first, followed by the name of the field.